config/class-site-details-index.php

```diff
@@ -202,7 +202,7 @@ public function get_site_details() {
 	public function put_site_details() {
 		$site_details = $this->get_site_details();
 
-		if ( defined( 'SERVICES_API_URL' ) && defined( 'SERVICES_AUTH_TOKEN' ) ) {
+		if ( defined( 'SERVICES_API_URL' ) && defined( 'SERVICES_AUTH_TOKEN' ) && ! empty( SERVICES_AUTH_TOKEN ) ) {
```

Comment:
Just to be extra safe should we check that `SERVICES_API_URL` is not empty as well?

Reply:
I thought about this... but this is a define that will be added to `vip-secrets.php`, so I think it should be quite reliable (i.e, an empty `SERVICES_API_URL` would be as bad as a wrong url on it)